src/main/drivers/io.c

```diff
@@ -238,19 +238,25 @@ void IOToggle(IO_t io)
 // claim IO pin, set owner and resources
 void IOInit(IO_t io, resourceOwner_e owner, uint8_t index)
 {
+    if (!io)
```

Comment:
Not sure here... Calling IOInit on NONE is bad practice, programmer should check this and handle it explicitly.

Other cases (Release + Owner) are fine